Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM alpine:3.3
-MAINTAINER jp@roemer.im
+FROM willemvd/ubuntu-unprivileged-git-ssh:latest
```

Comment:
This opens up a huge security hole and should not, in any way, be accepted by the maintainers of Gitea.